README.md

```diff
@@ -83,6 +83,10 @@ For [code execution](https://microsoft.github.io/autogen/docs/FAQ/#code-executio
 
 For LLM inference configurations, check the [FAQs](https://microsoft.github.io/autogen/docs/FAQ#set-your-api-endpoints).
 
+## Simple Example in ChatGPT Style
+
+[Sample code]((https://github.com/microsoft/autogen/blob/main/samples/simple_chat.py)) allowing the user to chat with an AutoGen agent in ChatGPT style
```

Comment:
Remove the section title. Move the example before line 126.

Reply:
Just to confirm, that movement would insert the new line right below the NVDA-TESLA graphic, and before the following line:

Please find more [code examples](https://microsoft.github.io/autogen/docs/Examples#automated-multi-agent-chat) for this feature.

Reply:
Yes. That was my suggestion. Change the wording to fit in.

Reply:
Then I would propose this wording for the inserted sentence:

Alternatively, the [sample code here](https://github.com/microsoft/autogen/blob/main/samples/simple_chat.py) allows a user to chat with an AutoGen agent in ChatGPT style.